incubator/busbar/templates/daemonset.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1beta1
+kind: DaemonSet
+metadata:
+  name: {{ template "kubeRegistry.fullname" . }}-internal
```

Comment:
Anything your chart installs should use `{{ template "busbar.fullname" . }}`. Use components to distinguish.